src/MockBot.Api/Controllers/ChatController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MockBot.Api.Interfaces;
+using MockBot.Api.Models;
+
+namespace MockBot.Api.Controllers;
+
+[Route("api/chats")]
+public class ChatController : Controller
+{
+    private readonly IChatService _chatService;
+
+    public ChatController(IChatService chatService)
+    {
+        _chatService = chatService;
+    }
+
+    [HttpGet]
+    public IEnumerable<Chat> FindAll()
+    {
+        return _chatService.FindAll();
+    }
+
+    [HttpGet("{id}")]
+    public Chat Get(Guid id)
+    {
+        return _chatService.Get(id);
+    }
+    
+    [HttpPost]
+    public Chat Post()
+    {
+        return _chatService.Create();
+    }
+    
+    [HttpGet("{chatId}/messages")]
+    public List<Message> GetMessages(Guid chatId)
```

Comment:
```suggestion
    public IEnumerable<Message> GetMessages(Guid chatId)
```
Would be nice to have this too

Reply:
I agree with this .... `IEnumerable` can always be cats to a `List` for consuming clients if necessary

Reply:
Alright, is it fine to initialise Messages on the Chat model as a Collection?